### PR TITLE
Mash out an mvp

### PR DESCRIPTION
**Pull request by petereast**

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
 import Dashboard from './dashboard/Dashboard';
+import Submission from './submission/Submission';
+import Navbar from './components-core/Navbar';
 
 const App: React.StatelessComponent = (): JSX.Element => (
-    <Router>
-        <Route exact path="/" component={(): JSX.Element => <div>home route</div>} />
-        <Route exact path="/dashboard" component={Dashboard} />
-    </Router>
+    <div>
```
Comment (diversemix) [inline client/src/App.tsx]: Why did we add the surrounding `div` ?

File: client/src/components-core/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+const NavBar = () => {
+    return (
+        <div style={containerStyle}>
+            <div
+                style={{
+                    display: 'flex',
+                    flex: '1 1 auto',
+                }}
+            >
+                <Link to="/" style={navBarItemStyle}>
```
Comment (diversemix) [inline client/src/components-core/Navbar.tsx]: Would it be worth making this a `class` and not a `style` ?

File: client/src/submission/submission.entities.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
+// This should contain the types and functions to call the graphql resolvers
+import { request } from 'graphql-request';
+
+//  TYPES
+
+export interface Submission {
```
Comment (diversemix) [inline client/src/submission/submission.entities.ts]: I'd suggest pulling types out into a separate file?

File: client/src/components-core/Navbar.tsx (hunk 2 above)
Comment (petereast) [reply client/src/components-core/Navbar.tsx]: Yes but we haven't configured webpack to load css yet

File: client/src/submission/submission.entities.ts (hunk 3 above)
Comment (petereast) [reply client/src/submission/submission.entities.ts]: Yep

**Pull request merged**